lib/trento/application/integration/discovery/discovery.ex

```diff
@@ -137,7 +137,13 @@ defmodule Trento.Integration.Discovery do
     ClusterPolicy.handle(event, current_cluster_id)
   end
 
-  defp do_handle(%{"discovery_type" => "saptune_discovery"} = event), do: HostPolicy.handle(event)
+  defp do_handle(%{"discovery_type" => "saptune_discovery", "agent_id" => agent_id} = event) do
```

Comment:
super nitpicky comment but I guess we could save a concatenation by checking the sum of the length of each line?